<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>705</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="codigo_fuente">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>271</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabla_analisis">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>581</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>SÃ­mbolo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Token</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Estado Act.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Estado Sig.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Validez</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="analizar_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Analizar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
